servers/rendering/renderer_rd/shaders/forward_clustered/scene_forward_clustered.glsl

```diff
@@ -1765,7 +1765,7 @@ void fragment_shader(in SceneData scene_data) {
 
 					pssm_coord /= pssm_coord.w;
 
-					shadow = sample_directional_pcf_shadow(directional_shadow_atlas, scene_data.directional_shadow_pixel_size * directional_lights.data[i].soft_shadow_scale * blur_factor, pssm_coord);
+					shadow = sample_directional_pcf_shadow(directional_shadow_atlas, scene_data.directional_shadow_pixel_size * directional_lights.data[i].soft_shadow_scale * (blur_factor+(1-blur_factor)*float(directional_lights.data[i].blend_splits)), pssm_coord);
```

Comment:
Please follow the surrounding code style (spaces around operators).

You can run clang-format on the .glsl files to apply it automatically.

Reply:
I've added the missing spaces